[2023-07-11T13:31:58.560+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:31:58.593+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:31:58.602+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-07-11T13:31:58.698+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_file> on 2023-07-01 00:00:00+00:00
[2023-07-11T13:31:58.714+0000] {standard_task_runner.py:57} INFO - Started process 6189 to run task
[2023-07-11T13:31:58.771+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'exercice_1', 'create_file', 'scheduled__2023-07-01T00:00:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/correction.py', '--cfg-path', '/tmp/tmphxkdn4ou']
[2023-07-11T13:31:58.802+0000] {standard_task_runner.py:85} INFO - Job 97: Subtask create_file
[2023-07-11T13:31:59.336+0000] {task_command.py:410} INFO - Running <TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [running]> on host 1d6232dcfa4d
[2023-07-11T13:31:59.660+0000] {abstractoperator.py:598} ERROR - Exception rendering Jinja template for task 'create_file', field 'bash_command'. Template: '/opt/***/dags/file/mon_script.sh'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 590, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 153, in render_template
    template = jinja_env.get_template(value)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/file/mon_script.sh
[2023-07-11T13:31:59.703+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1407, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1531, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2179, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 1254, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 590, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 153, in render_template
    template = jinja_env.get_template(value)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/file/mon_script.sh
[2023-07-11T13:31:59.831+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=exercice_1, task_id=create_file, execution_date=20230701T000000, start_date=20230711T133158, end_date=20230711T133159
[2023-07-11T13:31:59.928+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 97 for task create_file (/opt/airflow/dags/file/mon_script.sh; 6189)
[2023-07-11T13:31:59.986+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-11T13:32:00.137+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-11T13:41:59.613+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:41:59.648+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:41:59.650+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-07-11T13:41:59.709+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_file> on 2023-07-01 00:00:00+00:00
[2023-07-11T13:41:59.720+0000] {standard_task_runner.py:57} INFO - Started process 6482 to run task
[2023-07-11T13:41:59.733+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'exercice_1', 'create_file', 'scheduled__2023-07-01T00:00:00+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/correction.py', '--cfg-path', '/tmp/tmpbkabz_1a']
[2023-07-11T13:41:59.745+0000] {standard_task_runner.py:85} INFO - Job 137: Subtask create_file
[2023-07-11T13:41:59.948+0000] {task_command.py:410} INFO - Running <TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [running]> on host 1d6232dcfa4d
[2023-07-11T13:42:00.283+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='exercice_1' AIRFLOW_CTX_TASK_ID='create_file' AIRFLOW_CTX_EXECUTION_DATE='2023-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-01T00:00:00+00:00'
[2023-07-11T13:42:00.294+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-11T13:42:00.301+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '/opt/***/dags/file/mon_script.sh ']
[2023-07-11T13:42:00.340+0000] {subprocess.py:86} INFO - Output:
[2023-07-11T13:42:00.398+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-11T13:42:00.501+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=exercice_1, task_id=create_file, execution_date=20230701T000000, start_date=20230711T134159, end_date=20230711T134200
[2023-07-11T13:42:00.599+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-11T13:42:00.661+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-07-11T13:48:14.827+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:48:14.861+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:48:14.868+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-07-11T13:48:14.989+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_file> on 2023-07-01 00:00:00+00:00
[2023-07-11T13:48:15.000+0000] {standard_task_runner.py:57} INFO - Started process 6667 to run task
[2023-07-11T13:48:15.019+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'exercice_1', 'create_file', 'scheduled__2023-07-01T00:00:00+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/correction.py', '--cfg-path', '/tmp/tmp4l5heq9x']
[2023-07-11T13:48:15.035+0000] {standard_task_runner.py:85} INFO - Job 159: Subtask create_file
[2023-07-11T13:48:15.166+0000] {task_command.py:410} INFO - Running <TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [running]> on host 1d6232dcfa4d
[2023-07-11T13:48:15.404+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='exercice_1' AIRFLOW_CTX_TASK_ID='create_file' AIRFLOW_CTX_EXECUTION_DATE='2023-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-01T00:00:00+00:00'
[2023-07-11T13:48:15.410+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-11T13:48:15.415+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '/opt/***/dags/file/mon_script.sh ']
[2023-07-11T13:48:15.466+0000] {subprocess.py:86} INFO - Output:
[2023-07-11T13:48:15.670+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-11T13:48:15.757+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=exercice_1, task_id=create_file, execution_date=20230701T000000, start_date=20230711T134814, end_date=20230711T134815
[2023-07-11T13:48:16.007+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-11T13:48:16.134+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-07-11T13:51:14.119+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:51:14.158+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:51:14.159+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-07-11T13:51:14.344+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_file> on 2023-07-01 00:00:00+00:00
[2023-07-11T13:51:14.359+0000] {standard_task_runner.py:57} INFO - Started process 6790 to run task
[2023-07-11T13:51:14.387+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'exercice_1', 'create_file', 'scheduled__2023-07-01T00:00:00+00:00', '--job-id', '180', '--raw', '--subdir', 'DAGS_FOLDER/correction.py', '--cfg-path', '/tmp/tmpmz7oxouo']
[2023-07-11T13:51:14.395+0000] {standard_task_runner.py:85} INFO - Job 180: Subtask create_file
[2023-07-11T13:51:14.569+0000] {task_command.py:410} INFO - Running <TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [running]> on host 1d6232dcfa4d
[2023-07-11T13:51:14.762+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='exercice_1' AIRFLOW_CTX_TASK_ID='create_file' AIRFLOW_CTX_EXECUTION_DATE='2023-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-01T00:00:00+00:00'
[2023-07-11T13:51:14.766+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-11T13:51:14.768+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '/opt/***/dags/file/mon_script.sh ']
[2023-07-11T13:51:14.826+0000] {subprocess.py:86} INFO - Output:
[2023-07-11T13:51:14.927+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-11T13:51:15.028+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=exercice_1, task_id=create_file, execution_date=20230701T000000, start_date=20230711T135114, end_date=20230711T135115
[2023-07-11T13:51:15.112+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-11T13:51:15.369+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-11T13:53:35.588+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:53:35.610+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [queued]>
[2023-07-11T13:53:35.611+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-07-11T13:53:35.747+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_file> on 2023-07-01 00:00:00+00:00
[2023-07-11T13:53:35.773+0000] {standard_task_runner.py:57} INFO - Started process 6913 to run task
[2023-07-11T13:53:35.793+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'exercice_1', 'create_file', 'scheduled__2023-07-01T00:00:00+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/correction.py', '--cfg-path', '/tmp/tmpyh0957uk']
[2023-07-11T13:53:35.812+0000] {standard_task_runner.py:85} INFO - Job 202: Subtask create_file
[2023-07-11T13:53:35.984+0000] {task_command.py:410} INFO - Running <TaskInstance: exercice_1.create_file scheduled__2023-07-01T00:00:00+00:00 [running]> on host 1d6232dcfa4d
[2023-07-11T13:53:36.302+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='exercice_1' AIRFLOW_CTX_TASK_ID='create_file' AIRFLOW_CTX_EXECUTION_DATE='2023-07-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-01T00:00:00+00:00'
[2023-07-11T13:53:36.311+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-11T13:53:36.324+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '/opt/***/dags/file/mon_script.sh ']
[2023-07-11T13:53:36.349+0000] {subprocess.py:86} INFO - Output:
[2023-07-11T13:53:36.397+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-11T13:53:36.497+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=exercice_1, task_id=create_file, execution_date=20230701T000000, start_date=20230711T135335, end_date=20230711T135336
[2023-07-11T13:53:36.613+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-11T13:53:36.708+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
